derived/derivation.adoc

```diff
@@ -90,6 +90,11 @@ IDerived "0..*" -> "1..*"  Node: base
 Each derived node references one or more base nodes.
 Any (base) node can have none or more derived nodes.
 
+[horizontal,labelwidth=12]
+QUESTION:: Should a derived node also be able to refer to features inside a node? Or do we leave that for specific classes implementing `IDerived`? 
+QUESTION:: The assumption is that derived nodes can have children that are not derived nodes.
```

Comment:
Derived nodes can have any kind of features, including children that might or might not implement `IDerived`.